client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -65,6 +65,24 @@
 	text-overflow: ellipsis;
 }
 
+.domain-management-list__notice {
+	margin-bottom: 0;
+}
+
+.domain-management-list-item {
+	&.busy {
+		background-color: #f3f6f8;
+
+		.domain-management-list-item__title {
+			color: #bcc4ca;
```

Comment:
Here I just noticed that you were using `lighten( $gray, 20% )` for `.domain-management-list-item__meta`, but on the view it looked exactly like `#bcc4ca` in `.domain-management-list-item__title` above it (in the view). I didn't bother to check it with a color dropper or anything, just hoped it's actually the same :laughing: